src/utils/searchHelpers.ts

```diff
@@ -219,3 +219,10 @@ export const fundingSourceAutocompleteFilterOptions = (
     return identifier.includes(filter) || names.some((name) => name.includes(filter));
   });
 };
+
+// Note: The waiting time is a bit arbitrary, but it should be enough time for the reindexing to finish in many cases.
```

Comment:
viser seg at 5sek venting som her ikke er nok for første runde med reindeksering (i dev). Men etter første oppdatering går de påfølgende bra. Tenker dette er en OK trade-off for nå